<launch>
  <param name="ref_lat" value="39.98304881111"/>
  <param name="ref_lon" value="-111.9903776"/>
  <param name="ref_h" value="0"/>
  <param name="DROP_LOCATION" value="[INF,INF,INF]"/>
  <!--[99999.9,99999.9,99999.9]-->

  <node name="mission_planner" pkg="metis" type="mainPlanner.py" output = "screen" />
  <node name="fake_interop" pkg="metis" type="FakeInteropElberta.py" output = "screen" />
  <node name="plotter" pkg="metis" type="plotter.py" output="screen" />
  <node name="truth_throttle" type="throttle" pkg="topic_tools" args="messages /fixedwing/truth 1.0 /truth_slow" />
  <node name="state_convert" type="throttle" pkg="topic_tools" args="messages /fixedwing/truth 3.0 /state" />
  <node name="payload_drop" pkg="metis" type="payloadDrop.py" output="screen" />


</launch>
